backend/src/users/user-relationship/entities/user-relationship.entity.ts

```diff
@@ -15,25 +15,25 @@ export enum validRelationships {
 }
 
 @Entity('user_relationships')
-@Check(`"source_id" <> "target_id"`) // 				makes [2,2] impossible
+@Check(`"source" <> "target"`) // 				makes [2,2] impossible
```

Comment:
we can have multiple rows with the same source/target
which causes some problems
to partially solve this we can add:

```ts
@Unique(['source','target'])
```
which solves `[{2,3},{2,3}]` but not `[{2,3},{3,2}]`

Reply:
*kuch*

nevermind